spec/acceptance/counts_visits_for_real_data_spec.rb

```diff
@@ -0,0 +1,52 @@
+describe 'counts visits for real data' do
+  let(:file_gateway) { Gateway::File.new(path: './data/webserver.log') } 
```

Comment:
Acceptance specs are de-coupled from any delivery mechanism (they don't test `main.rb` script for example) but they do use the whole stack throughout, including real gateways.